yb-voyager/cmd/exportSchema.go

```diff
@@ -835,18 +834,14 @@ func removeRedundantIndexes(fileName string) ([]string, error) {
 	//using issues here as this GetRedundantIndexIssues already resolves the existing index to the correct final one so using it right now
 	//but once we remvoe the reporting of issues we can modify this function to resolve that and give a required map directly
 	//TODO: revisit this index object name check to properly done on each item of index qualified name instead of some formatted string.
-	redundantIssues := queryissue.GetRedundantIndexIssues(redundantIndexesInfo)
+	resolvedRedundantIndexes := utils.GetResolvedRedundantIndexes(redundantIndexesInfo)
```

Comment:
rather than utils, do you want to move this logic to the assessment layer, where you have a function to get the redundant indexes?